docs/modules/ROOT/pages/env_var_args.adoc

```diff
@@ -0,0 +1,56 @@
+
+=== PRODUCT_CONFIG
+
+*Default value*: `/etc/stackable/nifi-operator/config-spec/properties.yaml`
+
+*Required*: false
+
+*Multiple values:* false
+
+[source]
+----
+export PRODUCT_CONFIG=/foo/bar/properties.yaml
+cargo run -- run
```

Comment:
```suggestion
PRODUCT_CONFIG=/foo/bar/properties.yaml cargo run -- run
```

Reply:
I have it like this in all other operators, so i would like to keep it like this.